controller/controller.js

```diff
@@ -0,0 +1,5 @@
+Controller.extend({
+  changeAge(model, age) {
+    model.set('age', age);
```

Comment:
coding convention we use here is i_ for number based. 
We also should not name the storage val and input the same. 
model.set('i_age', age);